### ISSUE TITLE
[bug]: 6.0.2 retries successful uploads

### ISSUE THREAD

**Issue by superbryntendo** [pingdotgg/uploadthing#484]
### Provide environment information

```bash
System:
    OS: macOS 14.1.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 24.15 GB / 64.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 19.4.0 - ~/.nvm/versions/node/v19.4.0/bin/node
    Yarn: 3.6.1 - ~/.yarn/bin/yarn
    npm: 9.2.0 - ~/.nvm/versions/node/v19.4.0/bin/npm
  Browsers:
    Chrome: 112.0.5615.137
    Safari: 17.1
  npmPackages:
    @uploadthing/react: ^6.0.1 => 6.0.1 
    typescript: ^4.3.5 => 4.9.5 
    uploadthing: ^6.0.2 => 6.0.2
```

### Describe the bug

<img width="486" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/4926824/0cdbf0a7-0198-4b31-9ec0-424559e44479">

Images are getting uploaded successfully after updating to 6.0.2, but uploadthing seems to believe they're failing so it keeps retrying.

### Link to reproduction

(I haven't used stackblitz before, but the link is throwing an error)

### To reproduce

I'm using basically the default setup. Here's my fileRoute:

```
recipeImage: f({ image: { maxFileSize: '4MB' } }).onUploadComplete(
    async ({ metadata, file }) => {
      console.log('uploaded:', file.url, metadata)
    }
  ),
```

### 👨‍👧‍👦 Contributing

- [ ] 🙋‍♂️ Yes, I'd be down to file a PR fixing this bug!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by pauledrupt** [comment]
same here

**Comment by ugumerie** [comment]
same here too

**Comment by markflorkowski** [comment]
Does this also happen in production builds, or just local dev?

**Comment by pauledrupt** [comment]
> Does this also happen in production builds, or just local dev?

yes, this only happens in dev for me.

**Comment by markflorkowski** [comment]
This should be fixed in [6.0.3](https://github.com/pingdotgg/uploadthing/releases)
